src/main/java/com/spotify/dns/XBillDnsSrvResolver.java

```diff
@@ -22,15 +22,18 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.xbill.DNS.Lookup;
-import org.xbill.DNS.Record;
-import org.xbill.DNS.SRVRecord;
+import org.xbill.DNS.*;
```

Comment:
Are you allowed to use * imports? It's a discouraged practice and `org.xbill.DNS.Record` may cause you troubles in the future with wildcard-imports as `Record` became a reserved keyword in Java 14 (JEP 395).